masterlist.yaml

```diff
@@ -17921,6 +18202,7 @@ plugins:
     after:
       - 'Immersive Citizens - AI Overhaul.esp'
   - name: 'ELE_Legendary_Lite.esp'
+    global_priority: 77
```

Comment:
> (Also, https://loot.github.io/docs/contributing/Global-Priorities.html really ought to get updated. :/)

I thought I'd already done this when I standardised priority values across the masterlists... I'll have to dig out the forum post I made about it.